### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by jklaiber**
- Update links to use the new repos
- Update to version 1.3.2

### CODE REVIEW

File: docs/docs/getting-started.md [hunk 1]
```diff
@@ -17,29 +17,41 @@ To address the Gateway you can either compile the Protobuf files yourself or use
...
 ```bash
-$ go get github.com/jalapeno-api-gateway/protorepo-jagw-go@v1.3.0
+$ go get github.com/jalapeno-api-gateway/jagw-go@v1.3.2
+```
+
+#### Option 1.3: Python only
+
+If you are using Python, you will not find the pre-compiled files in the release page, but rather in a dedicated repository:
+
+- [JAGW-Python](https://github.com/jalapeno-api-gateway/jagw-python/releases/latest)
```
Comment (severindellsperger) [inline docs/docs/getting-started.md:38]: I suggest we're giving the link to the repo and not the empty releases:
```suggestion
- [JAGW-Python](https://github.com/jalapeno-api-gateway/jagw-python/)
```

**Review by severindellsperger — COMMENTED** [review]
Please address my suggestion.

File: docs/docs/getting-started.md (hunk 1 above)
Comment (jklaiber) [reply docs/docs/getting-started.md:38]: We will create a release. Therefore, I suggest leaving the link as it is.

**Review by severindellsperger — APPROVED** [review]

**Comment by severindellsperger** [comment]
LGTM.

**Pull request merged**